Packs/DelineaSS/ReleaseNotes/3_0_0.md

```diff
@@ -0,0 +1,13 @@
+
+#### Integrations
+
+##### DelineaSS
+- Update the implementation of the fetch-credentials.
+- Updated the pack with autoComment feature for commands.
+  - delinea-secret-rpc-changepassword
+  - delinea-secret-delete
+  - delinea-secret-password-update
+  - delinea-secret-get
+
+- Updated the Docker image to: *demisto/python3:3.10.11.59070*.
```

Comment:
```suggestion
- Update implementation of the ***fetch-credentials*** command.
- Added the *autoComment* argument to the following commands.
  - ***delinea-secret-rpc-changepassword***
  - ***delinea-secret-delete***
  - ***delinea-secret-password-update***
  - ***delinea-secret-get***

- Updated the Docker image to: *demisto/python3:3.10.11.59070*.
```

Reply:
@merit-maita I have addressed the comment. Please take a look